assets/i18n/en.json

```diff
@@ -176,6 +176,7 @@
     "I18N_FOOTER_TEACH_LEARN_ALL_CAPS": "TEACH/LEARN",
     "I18N_FOOTER_TEACH_PAGE": "For Parents/Teachers",
     "I18N_FOOTER_TERMS_OF_SERVICE": "Terms of Service",
+    "I18N_FOOTER_VOLUNTEER": "Volunteer",
```

Comment:
Should we have two keys one for footer and one for header even when they say the same thing? (and I think we expect them to stay the same).

Reply:
done